src/main/scala/gitbucket/core/controller/IndexController.scala

```diff
@@ -194,18 +195,62 @@ trait IndexControllerBase extends ControllerBase {
     contentType = formats("json")
     val user = params("user").toBoolean
     val group = params("group").toBoolean
+
+    val issueRe = (context.baseUrl + """/(\w+)/(\w+)/(?:issues|pull)/(\d+)""").r
+    val sortByFunc: Account => String = context.request.referrer match {
```

Comment:
Getting repository and issue (or pull request) identifier from referrer is nice idea, but I think it's little tricky. It might be better to pass these information as optional query string parameters.

Reply:
As I know, this request is called from `helper/attached.scala.html` through `helper/preview.scala.html`.  `preview.scala.html` called from issue/pulls comment, wiki edit and release message edit. So it is complicated to add optional query string.

Reply:
Ah... Okay. It wasn't so easy. Anyway, I don't prefer kind of ad-hoc solution. I will try to find an another way. Sorry for keeping this pull request reviewing.